tests/unit-tests/admin/test-class-sensei-setup-wizard.php

```diff
@@ -247,8 +267,6 @@ public function testActivationRedirectNoAdmin() {
 			->method( 'redirect_to_setup_wizard' );
 
 		$setup_wizard_mock->activation_redirect();
-
-		$this->assertNotFalse( get_transient( 'sensei_activation_redirect' ), 'Transient should not be removed' );
```

Comment:
The next test still mentions Transient even though we are no longer using transients, should we rename?

Reply:
Fixed with the https://github.com/Automattic/sensei/commit/2d7b8eb903dbb8dc00b466e502505cda51ce9c79